spec/features/editing_file_attachments/upload_file_attachment_spec.rb

```diff
@@ -30,15 +30,25 @@ def and_i_upload_a_file_attachment
     @asset_manager_request = stub_asset_manager_receives_an_asset(filename: @attachment_filename)
     @publishing_api_request = stub_publishing_api_put_content(@edition.content_id, {})
 
-    @attachment_filename = "attachment.csv"
+    @attachment_filename = "attachment.pdf"
     find('form input[type="file"]').set(Rails.root.join(file_fixture(@attachment_filename)))
     fill_in "title", with: "A title"
     click_on "Upload"
   end
 
   def then_i_can_see_the_attachment_markdown
-    expect(page).to have_content("[Attachment: attachment.csv]")
-    expect(page).to have_content("[AttachmentLink: attachment.csv]")
+    expect(page).to have_content("[Attachment: attachment.pdf]")
+  end
+
+  def and_i_can_see_a_preview_of_the_attachment_link
+    attachment = FileAttachment::Revision.last
+    file_size = number_to_human_size(attachment.byte_size)
+    number_of_pages = pluralize(attachment.number_of_pages, "page")
```

Comment:
I'd suggest we hard code these rather than having to use the helpers to look these up, as it has to dig a reasonable way into the code to pull out the values.

To make it clearer what they are they could put in the pdf file name. E.g. `10kb-2-page.pdf` ?

Reply:
I've renamed the PDF and hardcoded the values 👍